app/iCubSim/test_ft_right_leg.ini

```diff
@@ -0,0 +1,2 @@
+name "Test FT sensors Right Leg"
+portname /${robotname}/right_leg/analog:o
```

Comment:
The FT sensors are not present in the iCubSim ODE simulator, however are present in the real robot, so I think we should move this configuration file in the iCubTest config.